src/fields/lib/abstract-field.js

```diff
@@ -75,6 +75,7 @@ module.exports = class AbstractField {
     if (this.isWritableOnce) this.isRequired = true;
     if (this.isRequired) this.isWritable = true;
     if (this.isPrimaryKey && !options.hasOwnProperty('isFilterable')) this.isFilterable = true;
+    if (this.isPrimaryKey && !this.isFilterable) console.warn(`isPrimaryKey implies isFilterable for key ${this.key}`);
```

Comment:
Please use `Restypie.Logger.warn()` instead. Hmm and we'll also need to make the level configurable : https://github.com/SylvainEstevez/Restypie/blob/master/src/logger.js#L9 (i can do this later)